apps/qubit/modules/informationobject/templates/_cardViewResults.php

```diff
@@ -11,15 +11,15 @@
     <?php endif; ?>
 
       <a href="<?php echo url_for(array('module' => 'informationobject', 'slug' => $doc['slug'])) ?>">
-        <?php if (!empty($doc['digitalObject']['thumbnailPath'])
+        <?php if (isset($doc['digitalObject']) && !empty($doc['digitalObject']['thumbnailPath'])
```

Comment:
If I leave it as it was I get the following notice:

```[Mon Jul 03 17:18:58.160718 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP Notice:  Undefined index: digitalObject in /home/radda/www/atom-2/vendor/symfony/lib/escaper/sfOutputEscaperArrayDecorator.class.php on line 119, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160724 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP Stack trace:, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160728 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP   1. {main}() /home/radda/www/atom-2/qubit_dev.php:0, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160731 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP   2. sfContext->dispatch() /home/radda/www/atom-2/qubit_dev.php:21, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160734 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP   3. sfFrontWebController->dispatch() /home/radda/www/atom-2/vendor/symfony/lib/util/sfContext.class.php:170, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160741 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP   4. sfController->forward() /home/radda/www/atom-2/vendor/symfony/lib/controller/sfFrontWebController.class.php:48, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160744 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP   5. sfFilterChain->execute() /home/radda/www/atom-2/vendor/symfony/lib/controller/sfController.class.php:238, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160747 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP   6. QubitTransactionFilter->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfFilterChain.class.php:53, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160750 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP   7. sfFilterChain->execute() /home/radda/www/atom-2/lib/filter/QubitTransactionFilter.class.php:48, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160753 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP   8. sfHistoryPluginFilter->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfFilterChain.class.php:53, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160756 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP   9. sfFilterChain->execute() /home/radda/www/atom-2/plugins/sfHistoryPlugin/lib/sfHistoryPluginFilter.class.php:18, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160759 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  10. sfRenderingFilter->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfFilterChain.class.php:53, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160762 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  11. sfFilterChain->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfRenderingFilter.class.php:33, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160765 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  12. QubitSettingsFilter->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfFilterChain.class.php:53, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160767 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  13. sfFilterChain->execute() /home/radda/www/atom-2/lib/filter/QubitSettingsFilter.class.php:64, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160770 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  14. QubitLimitIpFilter->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfFilterChain.class.php:53, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160772 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  15. sfFilterChain->execute() /home/radda/www/atom-2/lib/filter/QubitLimitIp.class.php:39, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160775 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  16. QubitSslRequirementFilter->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfFilterChain.class.php:53, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160778 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  17. sfFilterChain->execute() /home/radda/www/atom-2/lib/filter/QubitSslRequirementFilter.class.php:35, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160780 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  18. QubitMeta->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfFilterChain.class.php:53, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160785 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  19. sfFilterChain->execute() /home/radda/www/atom-2/lib/filter/QubitMeta.class.php:31, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160787 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  20. QubitLimitResults->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfFilterChain.class.php:53, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160790 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  21. sfFilterChain->execute() /home/radda/www/atom-2/lib/filter/QubitLimitResults.class.php:34, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160793 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  22. sfExecutionFilter->execute() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfFilterChain.class.php:53, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160795 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  23. sfExecutionFilter->handleView() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfExecutionFilter.class.php:47, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160798 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  24. sfExecutionFilter->executeView() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfExecutionFilter.class.php:116, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160801 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  25. sfPHPView->render() /home/radda/www/atom-2/vendor/symfony/lib/filter/sfExecutionFilter.class.php:155, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160804 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  26. sfPHPView->renderFile() /home/radda/www/atom-2/vendor/symfony/lib/view/sfPHPView.class.php:185, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160806 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  27. require() /home/radda/www/atom-2/vendor/symfony/lib/view/sfPHPView.class.php:75, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160809 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  28. get_partial() /home/radda/www/atom-2/apps/qubit/modules/informationobject/templates/browseSuccess.php:338, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160812 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  29. sfPartialView->render() /home/radda/www/atom-2/vendor/symfony/lib/helper/PartialHelper.php:220, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160814 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  30. sfPHPView->renderFile() /home/radda/www/atom-2/vendor/symfony/lib/view/sfPartialView.class.php:124, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160817 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  31. require() /home/radda/www/atom-2/vendor/symfony/lib/view/sfPHPView.class.php:75, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse
[Mon Jul 03 17:18:58.160820 2017] [:error] [pid 30542] [client 192.168.56.1:55952] PHP  32. sfOutputEscaperArrayDecorator->offsetGet() /home/radda/www/atom-2/vendor/symfony/lib/view/sfPHPView.class.php:14, referer: http://192.168.56.128:81/atom-2/qubit_dev.php/informationobject/browse```

Maybe it has something to be with how the sfOutputEscaperArrayDecorator works.